NAMESPACE

```diff
@@ -222,11 +224,13 @@ export(tbl_butcher)
 export(tbl_continuous)
 export(tbl_cross)
 export(tbl_custom_summary)
+export(tbl_filter)
```

Comment:
We reserve the `tbl_*()` prefix for generating new tables.

Let's name the functions `filter_hierarchical` and `sort_hierarchical()`. Let's just make them functions instead of generics.

Reply:
Done!